kuma/wiki/tests/test_views.py

```diff
@@ -1190,6 +1190,22 @@ def test_create_on_404(self):
                                            locale=locale))
         eq_(302, response.status_code)
 
+    def test_creating_child_of_redirect(self):
+        """While try to create a child of a redirect,
+        the parent of the child should be redirect's parent"""
+        self.client.login(username='admin', password='testpass')
+        rev = revision(is_approved=True, save=True)
+        doc = rev.document
+        child_full_slug = doc.slug + "/" + "children_document"
+        doc._move_tree(new_slug="moved_doc")
```

Comment:
I am planning to change the functionality of moving mage aka redirect page in future. The way its implemented seems to be very strange both kuma and kitsune (Actually, its implemented badly in kitsune, that did not change in kuma era)
So therefore I am thinking to make the redirect with _move_tree
So in future if we change the functionality, the test will not break